php/php-codesniffer-standard/VIISON/Sniffs/Methods/UnitTestMethodNameSniff.php

```diff
@@ -0,0 +1,161 @@
+<?php
+namespace VIISON\StyleGuide\PHPCS\Standards\VIISON\Sniffs\Methods;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+/**
+ * Finds methods whos method name does not match their testdox description.
+ */
+class UnitTestMethodNameSniff implements Sniff
+{
+    const METHOD_NAME_DOES_NOT_EQUAL_TESTDOX = 'Unit test method "%s" does not equal testdox. Expected "%s"';
+
+    /**
+     * @inheritdoc
+     */
+    public function register()
+    {
+        return [
+            T_DOC_COMMENT_OPEN_TAG,
+        ];
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function process(File $phpcsFile, $stackPointer)
+    {
+        $tokens = $phpcsFile->getTokens();
+        $testdox = $this->getTestdox($tokens, $stackPointer);
+        if ($testdox === null) {
+            return;
+        }
+
+        $actualMethodName = $this->getMethodName($tokens, $stackPointer);
+        if ($actualMethodName === null) {
+            return;
+        }
+
+        $expectedMethodName = $this->getExpectedMethodNameFromTestdox($testdox);
+        if ($actualMethodName === $expectedMethodName) {
+            return;
+        }
+
+        $fixMethodName = $phpcsFile->addFixableError(sprintf(
+            self::METHOD_NAME_DOES_NOT_EQUAL_TESTDOX,
+            $actualMethodName,
+            $expectedMethodName
+        ), $stackPointer, 'EqualsTestdox');
+
+        if ($fixMethodName === true) {
+            $phpcsFile->fixer->beginChangeset();
+            $phpcsFile->fixer->replaceToken($stackPointer, $expectedMethodName);
+            $phpcsFile->fixer->endChangeset();
+        }
+    }
+
+    private function getMethodName($tokens, &$index)
+    {
+        $expectedTokens = [
+            'T_PUBLIC',
+            'T_FUNCTION',
+        ];
+
+        while (count($tokens) > $index) {
```

Comment:
This kind of breaks with convention.

```suggestion
        while ($index < count($tokens)) {
```